src/cdk/a11y/key-manager/list-key-manager.spec.ts

```diff
@@ -150,6 +150,7 @@ describe('Key managers', () => {
 
         keyManager.setActiveItem(0);
         itemList.reset([new FakeFocusable('zero'), ...itemList.toArray()]);
+        itemList.notifyOnChanges();
```

Comment:
Why is this manually calling `notifyOnChanges`? IIRC, this wasn't meant to be part of the public API of `QueryList`; tests would generally want to cause change detection instead (though it's possible I'm missing something here)

Reply:
It does `notifyOnChanges` instead of change detection because there is no test fixture. This test file doesn't create any kind of test component or use TestBed at all. It's vanilla test, so change detection is not available.